### ISSUE TITLE
Ea desktop 

### ISSUE THREAD

**Issue by Vicesimus** [Mixaill/awesome-gog-galaxy#115]
Hello. When you use xbox app with xbox game pass you have ea play pass. But ea play work only with ea desktop app and origin integration dont show game from ea desktop lib. It is reason why i ask about ea desktop integration.

**Comment by ciaron88** [comment]
YES!!! Please Please Please create an EA Desktop integration PLEASE!!!
